src/features/ShareRoom/page/category/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { Row, Col, Space } from 'antd';
+import TagsOutlined from '@ant-design/icons';
+import './../../index.scss';
+
+class Category extends React.Component{
+    render() {
+    return(
+        <div style={{marginTop:20}}>
+            <Row>
+                <Col><h1>Danh mục cho thuê</h1></Col>
+            </Row>
+            <div className='category'>
+            <Space direction="vertical">
+                <Row>
+                    <Space>
+                    <Col >
+                    <TagsOutlined style={{ fontSize: 20, color:'#8bb200' }} />
+                    </Col>
+                    <Col>
+                        <Link  className="name" to='/' style={{borderColor:'#8bb200'}} >Cho thuê phòng trọ</Link>
```

Comment:
can using category__item to replace for 'name'

Reply:
Research tag in antd and application for this component
here is e.g https://codepen.io/pen/?&editors=001